src/Microsoft.DotNet.Interactive/Connection/KernelCommandAndEventReceiver.cs

```diff
@@ -69,7 +69,7 @@ public KernelCommandAndEventReceiver(ReadCommandOrEvent readCommandOrEvent)
                 {
                     Log.Error(exception);
 
-                    return new CommandOrEvent(new DiagnosticLogEntryProduced(exception.Message, KernelCommand.None), true);
+                    return new CommandOrEvent(new ErrorProduced(exception.Message, KernelCommand.None), isParseError: true);
```

Comment:
I'm not entirely sure if using `ErrorProduced` for this is ok. I was not able to figure out another way to handle this (other than allowing the exception to propagate). Please let me know in case you think this is not ok.